### PR TITLE
Remove examples that show insecure practices

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #10342.

(Hide whitespace changes.)

### CODE REVIEW

File: snippets/csharp/VS_Snippets_ADO.NET/DataWorks SqlConnectionStringBuilder.Remove/CS/source.cs [hunk 1]
```diff
@@ -10,38 +10,29 @@ static void Main()
         try
         {
             string connectString =
-                "Data Source=(local);User ID=ab;Password= a1Pass@@11;" +
+                "Data Source=(local);User ID=ab;Password=myPassw0rd;" +
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_ADO.NET/DataWorks SqlConnectionStringBuilder.Remove/CS/source.cs:13]: I don't understand this change. Isn't this still bad?

**Review by BillWagner — APPROVED** [review]
This looks great @gewarren 

I had one question, primarily for my knowledge, Go ahead and :shipit: when ready.

File: snippets/csharp/VS_Snippets_ADO.NET/DataWorks SqlConnectionStringBuilder.Remove/CS/source.cs (hunk 1 above)
Comment (Rick-Anderson) [reply snippets/csharp/VS_Snippets_ADO.NET/DataWorks SqlConnectionStringBuilder.Remove/CS/source.cs:13]: How about changing it to `SecureGetPassword()` ? That code is too easy to copy and paste. Customer can possibly use AKV to the the secret. On Prem customers can do something more secure.

**Review by Rick-Anderson — COMMENTED** [review]
Massive improvement

**Pull request merged**